bin/dax_tools/GenerateProcessorTemplate

```diff
@@ -141,8 +142,8 @@ class Processor_{name}(ScanProcessor):
 
 SESSION_LEVEL_TEMPLATE = DEFAULT_TEMPLATE+"""
 # Format for the spider command line
-SPIDER_FORMAT = '''python {{spider}} -v --spm {{spm}} \
--p {{proj}} -s {{subj}} -e {{sess}} -d {{dir}} --suffix "{{suffix_proc}}"'''
+SPIDER_FORMAT = '''python {{spider}} -p {{proj}} -s {{subj}} -e {{sess}}\
```

Comment:
It was when I created this for the dax-spider package. It should not be there. 

Ben

> On Oct 2, 2015, at 20:00, aplassard <notifications@github.com> wrote:
> 
> In bin/dax_tools/GenerateProcessorTemplate:
> 
> > @@ -141,8 +142,8 @@ class Processor_{name}(ScanProcessor):
> >  
> >  SESSION_LEVEL_TEMPLATE = DEFAULT_TEMPLATE+"""
> >  # Format for the spider command line
> > -SPIDER_FORMAT = '''python {{spider}} -v --spm {{spm}} \
> > --p {{proj}} -s {{subj}} -e {{sess}} -d {{dir}} --suffix "{{suffix_proc}}"'''
> > +SPIDER_FORMAT = '''python {{spider}} -p {{proj}} -s {{subj}} -e {{sess}}\
> I'm just curious and I would check this if I was in front of a computer, but how is path settings handled? Is it part of the dax package or is it a python settings file like the grid engine file?
> …
> —
> Reply to this email directly or view it on GitHub.
>